src/App.jsx

```diff
@@ -360,6 +356,12 @@ useEffect(()=>{
             ))}
           </div>
         )}
+        {!loadingPosts &&
+          (user ? (
+          <ImgUpload user={user} snackBar={openSnackBar} />
```

Comment:
yes remove this and let line 240-253 of original stay , it seems like another bug

Reply:
I already made the changes you requested. Is it as expected? or some more changes are required?